lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1204,6 +1225,63 @@ void MfmaEncodingAttr::print(AsmPrinter &printer) const {
   printer << "}>";
 }
 
+//===----------------------------------------------------------------------===//
+// WMMA encoding
+//===----------------------------------------------------------------------===//
+
+Attribute WmmaEncodingAttr::parse(AsmParser &parser, Type type) {
+  if (parser.parseLess().failed())
```

Comment:
nit: even wen the patch is adding very basic things like that it is usually a good practice to add a simple test that would test the IR round trip. (ex: test/Conversion/triton_ops.mlir)